source/utils/timed-out.ts

```diff
@@ -149,11 +151,17 @@ export default (request: ClientRequest, delays: Delays, options: any) => {
 	});
 
 	if (delays.response !== undefined) {
-		request.once('upload-complete', (): void => {
-			const cancelTimeout = addTimeout(delays.response!, timeoutHandler, 'response');
+		request.once('upload-complete', () => {
+			const cancelTimeout = addTimeout(delays.response, timeoutHandler, 'response');
 			request.once('response', cancelTimeout);
 		});
 	}
 
 	return cancelTimeouts;
 };
+
+declare module 'http' {
+	interface ClientRequest {
+		[reentry]: boolean;
+	}
+}
```

Comment:
I don't think we should declare things that are internal to Got. Or is there a benefit of making this global?

Reply:
I did this to prevent needing to cast as any/unknown beforehand, but it wouldn't make a difference anyways due to it being a symbol